bobtask/input.go

```diff
@@ -42,6 +50,8 @@ func (t *Task) filteredInputs() ([]string, error) {
 		}
 	}()
 
+	t.InputDirty = fmt.Sprintf("%s\n%s", t.InputDirty, defaultIgnores)
```

Comment:
Sure we should add this to `InputDirty`? I think it's better to keep `InputDirty` as raw user input and not change it.

Reply:
I moved this from [here](https://github.com/benchkram/bob/pull/210/files#diff-93b66165514b0b1129265b30a1495ff242070cfcfce14e4ffb812d5cf658f0e2L154).

I think in our call we said to move it here, but if there's a more suitable place where I can move it can do.